presto-docs/src/main/sphinx/security/user-mapping.rst

```diff
@@ -41,6 +41,13 @@ following fields:
 * ``user`` (optional): replacement string to substitute against pattern.
   The default value is ``$1``.
 * ``allow`` (optional): boolean indicating if the authentication should be allowed.
+* ``case`` (optional): one of:
```

Comment:
Thanks for documenting this. Some comments:

* List items need to be indented two spaces. The extra indentation causes it to be treated as a quoted block.
* Use `*` for list items to be consistent with the other list items (above).
* The "default value" text below is hard to see, since it's not aligned with the list. We can make it part of the first list item.
* e.g. means "for example" and thus should have a comma afterwards
* The lower/upper items are long enough to wrap in the default HTML rendering, which makes them harder to read. We can shorten them a bit so that they don't wrap.

Here's the result of the above changes:

```
* ``case`` (optional): one of:

  * ``keep`` - keep matched user name as is (default behavior)
  * ``lower`` - lowercase matched user name, e.g., ``Admin`` or ``ADMIN`` will become ``admin``
  * ``upper`` - uppercase matched user name, e.g., ``admin`` or ``Admin`` will become ``ADMIN``
```

Rendered HTML:

<img width="741" alt="Screen Shot 2020-08-14 at 9 46 21 AM" src="https://user-images.githubusercontent.com/9230/90273245-8710ee00-de13-11ea-8fff-7fbf15951585.png">

Reply:
done